bakerydemo/static/css/main.css

```diff
@@ -385,6 +385,24 @@ li.has-submenu a.allow-toggle {
 .caret-custom:after {
     content: "▼"!important;
 }
+/* Mobile menu styling */
+@media (max-width: 768px) {
+  .nav-pills>li {
+    display: block;
+    float: none;
+    width: 100%!important;
```

Comment:
The `!important` shouldn't be necessary. Just change the `.nav-pills>li.breads` (and others) width definitions to only use the class name, not element, and then change this declaration to target `.nav-pills > .presentation`.